### PR TITLE
Improve plugin-help formatting by allowing text sections to include HTML.

### PR DESCRIPTION
**Pull request by cjwagner**
Letting plugins include HTML in most help fields seems to be the easiest way to support flexible formatting and makes it easy to include links and images.

/cc @rmmh

### CODE REVIEW

**Comment by cjwagner** [comment]
PTAL, the formatting is currently broken.

File: prow/plugins/releasenote/releasenote.go [hunk 1]
```diff
@@ -82,11 +82,11 @@ func init() {
...
-There are three valid types of release notes that can replace this label:
-	- PRs with a normal release note in the 'releasenote' block are given the label '` + releaseNote + `'.
-	- PRs that have a release note of 'none' in the block are given the label '` + releaseNoteNone + `' to indicate that the PR does not warrant a release note.
-	- PRs that contain 'action required' in their 'releasenote' block are given the label '` + releaseNoteActionRequired + `' to indicate that the PR introduces potentially breaking changes that necessitate user action before upgrading to the release.
-To support old behavior, this plugin also provides a '/release-note-none' command that can be used by organization members to specify that no release note is needed for the PR as an alternative to setting the 'releasenote' block contents to 'none'.`,
+<br>There are three valid types of release notes that can replace this label:
+<br>&nbsp&nbsp&nbsp&nbsp- PRs with a normal release note in the 'releasenote' block are given the label '` + releaseNote + `'.
```
Comment (stevekuznetsov) [inline prow/plugins/releasenote/releasenote.go]: Why not `<ul>` and `<li>` here?

File: prow/plugins/sigmention/sigmention.go [hunk 2]
```diff
@@ -62,8 +62,8 @@ func helpProvider(config *plugins.Configuration, enabledRepos []string) (*plugin
 	// Only the Description field is specified because this plugin is not triggered with commands and is not configurable.
 	return &pluginhelp.PluginHelp{
 			Description: `The sigmention plugin responds to SIG (Special Interest Group) Github team mentions like '@kubernetes/sig-testing-bugs'. The plugin responds in two ways:
-	1) The appropriate 'sig/*' and 'kind/*' labels are applied to the issue or pull request. In this case 'sig/testing' and 'kind/bug'.
-	2) If the user who mentioned the Github team is not a member of the organization that owns the repository the bot will create a comment that repeats the mention. This is necessary because non-member mentions do not trigger Github notifications.`,
+<br>&nbsp&nbsp&nbsp&nbsp1) The appropriate 'sig/*' and 'kind/*' labels are applied to the issue or pull request. In this case 'sig/testing' and 'kind/bug'.
```
Comment (stevekuznetsov) [inline prow/plugins/sigmention/sigmention.go]: `<ol>`?

**Pull request merged**